EIPS/eip-wrapped_token.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: <to be assigned>
+title: Wrapped token inspection
+author: David Mihal (@dmihal)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): ERC
+created: 2020-08-05
+---
+
+## Simple Summary
+Defines a standard interface for viewing the addresses and balances of wrapped tokens.
+
+## Abstract
+Many projects have emerged that create new assets by wrapping existing tokens. However, it can be difficult for applications to easily query which tokens are wrapped, and how much of each token is associated with the holder of the wrapper token.
+
+This EIP introduces a simple interface for exposing the addresses of all wrapped tokens, as well as querying wrapped token balances.
+
+## Motivation
+Many projects, particularly in decentralized finance, have created new assets by wrapping existing tokens. This includes, but is not limited to:
+
+* Liquidity pool tokens for automated market makers (Uniswap, Balancer)
+* Interest-bearing lending tokens (Compound cTokens, Aave aTokens)
+* Managed investment tokens (Set Protocol, yEarn vaults)
+* Utility wrapper tokens (DeFi777)
+
+While these tokens enable many new applications and financial primitives, they present a number of challenges for wallets and other applications. 
+
+### Example
+
+For example, if a user deposits tokens in a Uniswap pool, they will receive liquidity provider (LP) tokens. The user’s balance of these LP tokens does not provide information about the pooled tokens, so the user can not know how many funds are held in the pool without visiting the Uniswap dapp.
```

Comment:
Same comment as above, recommend removing the reference to Uniswap here and instead generalize this statement.